lib/git_reflow/commands/review.rb

```diff
@@ -5,16 +5,45 @@
   c.arg_name '[destination_branch] - the branch you want to merge your feature branch into'
   c.flag [:t, :title], default_value: 'last commit message'
   c.flag [:m, :message], default_value: 'title'
+  c.switch [:e, :edit]
   c.action do |global_options,options,args|
-    review_options = {
-      'base' => args[0],
-      'title' => global_options[:title],
-      'body' => global_options[:message]
-    }
 
-    review_options['title'] ||=  GitReflow.get_first_commit_message
-    review_options['body']  ||=  review_options['title']
+    commit_msg_file = '/tmp/git_reflow_pr_msg'
 
+    if global_options[:title] || global_options[:message]
+      review_options = {
+        'base' => args[0],
+        'title' => (global_options[:title]   || GitReflow.current_branch),
+        'body' =>  (global_options[:message] || GitReflow.get_first_commit_message)
+      }
+      editor = false
+    else # if edit flag or nothing is provided, open editor
+      editor = true
+    end
+
+    if options[:edit] # if edit we inject title & body into file before opening
+      File.open(commit_msg_file, 'w') do |file|
+        file.write(GitReflow.current_branch)
+        file.write("\n")
+        file.write(GitReflow.get_first_commit_message)
+      end
+    end
+
+    if editor
+      system('nano', commit_msg_file)
```

Comment:
would love to - but i couldn't find a solution that worked. got an example i can run with?